### PR TITLE
Feature/Implemented block with reviews

### PR DESCRIPTION
**Pull request by MickhaelSh**
Added AppRating component, CommentsBlock, Comment, added i18n to storybook, added AppRating to storybook
image
image
image
image

### CODE REVIEW

File: src/components/app-rating/AppRating.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <Box sx={ [styles.root, optionalStyles] }>
+      { bigNumberVariant }
+      { !(bigNumber && mobile) && (
+        <Rating
+          emptyIcon={ <StarSharp fontSize='inherit' style={ styles.emptyIcon } /> }
+          name='feedback'
+          precision={ 1 }
+          sx={ { typography: starsSize[size],
+            '& .MuiRating-icon': {
+              mx: spacing
```
Comment (Tolik170) [inline src/components/app-rating/AppRating.jsx:37]: As i see you dont pass spacing props anywhere, maybe better to remove it and move this sx to styles or we will need it in future?

File: src/components/comment/Comment.styles.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      border: '1px solid',
+      borderRadius: '50%',
+      borderColor: 'primary.900'
+    },
+    wrapper: {
+      display: 'flex',
+      flexDirection: 'column',
+      gap: '4px',
+      alignItems: 'start',
+      mb: 0,
```
Comment (Tolik170) [inline src/components/comment/Comment.styles.js:33]: you use **"m"** one line below, i guess you don`t need **"mb"**

Comment (MickhaelSh) [reply src/components/comment/Comment.styles.js:33]: no, ImageTitleWithDescriptions has m and mb in default styles, so to overnight them I should use both

File: src/containers/tutor-profile/coments-block/constants.js [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+        updatedAt: '2023-03-02T12:12:56.598Z',
+        __v: 0
+      },
+      createdAt: '2023-03-02T19:13:04.074Z',
+      updatedAt: '2023-03-02T19:13:04.074Z',
+      __v: 0
+    },
+    {
+      _id: '6400f540307bdcc5da14aa5a',
+      comment: 'not great at all, i teached her english at our lesson',
```
Comment (wonderflu) [inline src/containers/tutor-profile/coments-block/constants.js:126]: i am not trying to say anything, but it's really scary when such students TEACHED anyone :D

File: src/containers/tutor-profile/coments-block/constants.js [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+      author: {
+        _id: '63f905d3237ccffcf95d88da',
+        role: ['student'],
+        firstName: 'Tart',
+        lastName: 'Drilling',
+        photo:
+          'https://media.npr.org/assets/img/2014/08/07/monkey-selfie_custom-7117031c832fc3607ee5b26b9d5b03d10a1deaca-s300-c85.webp',
+        email: 'tartdrilling@gmail.com',
+        categories: [],
+        totalReviews: 0,
```
Comment (wonderflu) [inline src/containers/tutor-profile/coments-block/constants.js:17]: remove those fields.
there will be no totalReviews and averageRating

File: src/containers/tutor-profile/coments-block/constants.js [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
```
Comment (wonderflu) [inline src/containers/tutor-profile/coments-block/constants.js:137]: here the same

File: src/pages/tutor-profile/constants.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
+export const responseMock = {
+  user: {
+    _id: '6406eec81826f1e46fb6e05c',
+    role: ['student', 'tutor'],
+    firstName: 'ffffff',
+    lastName: 'fffff',
+    email: 'nataha-backend-queen@gmail.com',
+    categories: [],
+    totalReviews: 0,
```
Comment (wonderflu) [inline src/pages/tutor-profile/constants.js:9]: remove those

File: src/containers/tutor-profile/coments-block/constants.js (hunk 4 above)
Comment (MickhaelSh) [reply src/containers/tutor-profile/coments-block/constants.js]: fixed

File: src/containers/tutor-profile/coments-block/constants.js (hunk 5 above)
Comment (MickhaelSh) [reply src/containers/tutor-profile/coments-block/constants.js]: fixed

File: src/pages/tutor-profile/constants.js (hunk 6 above)
Comment (MickhaelSh) [reply src/pages/tutor-profile/constants.js:9]: fixed

**Review by wonderflu — APPROVED** [review]

**Review by Yurenko — APPROVED** [review]

File: src/components/app-rating/AppRating.jsx [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+import StarSharp from '@mui/icons-material/StarSharp'
+import Box from '@mui/material/Box'
+import Rating from '@mui/material/Rating'
+import Typography from '@mui/material/Typography'
+
+import { styles } from '~/components/app-rating/AppRating.styles'
+
+const starsSize = { small: 'body2', medium: 'body1', large: 'h6' }
+
+const AppRating = ({ value = '', numberVariant , size = 'small', withBackground, mobile, reviews, spacing = 1, ...props }) => {
```
Comment (maarekua) [inline src/components/app-rating/AppRating.jsx:11]: 8 props it toooo much.

File: src/components/app-rating/AppRating.jsx [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+  const { t } = useTranslation()
+
+  const optionalStyles = {
+    backgroundColor: withBackground && 'primary.50',
+    flexDirection: numberVariant === 'big' && 'column'
+  }
+
+  const smallnumberVariant = numberVariant === 'small' && (
```
Comment (maarekua) [inline src/components/app-rating/AppRating.jsx:19]: We can just assign the template, and then use ternary operator.

File: src/components/comment/Comment.jsx [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+  const timestamp = new Date(createdAt).toLocaleDateString('en-US', { year: 'numeric', month: 'long', day: 'numeric' })
+
+  const createNamePart = (value) => (
+    <Typography component={ 'span' } variant='body2'>
+      { value }
+    </Typography>
+  )
+
+  const userName = (
```
Comment (maarekua) [inline src/components/comment/Comment.jsx:20]: Too complicated for username

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/components/app-rating/AppRating.jsx (hunk 7 above)
Comment (MickhaelSh) [reply src/components/app-rating/AppRating.jsx]: fixed

File: src/components/app-rating/AppRating.jsx (hunk 8 above)
Comment (MickhaelSh) [reply src/components/app-rating/AppRating.jsx]: that value can be undefined, so i change to two variables

File: src/components/comment/Comment.jsx (hunk 9 above)
Comment (MickhaelSh) [reply src/components/comment/Comment.jsx:14]: fixed

**Review by maarekua — APPROVED** [review]

**Pull request merged**